### DISCUSSION TITLE
I didn't get my email/invoice

### DISCUSSION THREAD

**Discussion by deerdesignerdevops** [aclap-dev/video-downloadhelper#1371] [Help and Q&A]
### Your question

I bought the premium, got the license number in the page but didn't get the email confirming it nor the invoice. How do I get this?

### Addon & CoApp version and details

version: 9.1.1.0
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"e1096dcf8feb406b"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 28,
    "successfull_dl": 9,
    "download_history": {
        "__serializer_tag": "map",
        "__serializer_value": [
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_7139754299143915"
                    },
                    {
                        "download_result": {
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\Thiago\\dwhelper"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Congresso de Redes Sociais Hotmart Club.mp4"
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\Thiago\\dwhelper\\Congresso de Redes Sociais Hotmart Club.mp4"
                            },
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.congresso-redessociais.com/en/club/congresso-redessociais/products/4530907/content/14oYX99M7p"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1729170015426
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_1921118659041875"
                    },
                    {
                        "download_result": {
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\Thiago\\dwhelper"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Congresso de Redes Sociais Hotmart Club.mp4"
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\Thiago\\dwhelper\\Congresso de Redes Sociais Hotmart Club.mp4"
                            },
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.congresso-redessociais.com/en/club/congresso-redessociais/products/4530907/content/V4VdlPPE72"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1729170540680
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_2217213630848339"
                    },
                    {
                        "download_result": {
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\Thiago\\dwhelper"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Congresso de Redes Sociais Hotmart Club.mp4"
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\Thiago\\dwhelper\\Congresso de Redes Sociais Hotmart Club.mp4"
                            },
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.congresso-redessociais.com/en/club/congresso-redessociais/products/4530907/content/97BdnjBG4p"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1729171978832
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_2743810573877791"
                    },
                    {
                        "download_result": {
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\Thiago\\dwhelper"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Congresso de Redes Sociais Hotmart Club-01.mp4"
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\Thiago\\dwhelper\\Congresso de Redes Sociais Hotmart Club-01.mp4"
                            },
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.congresso-redessociais.com/en/club/congresso-redessociais/products/4530907/content/oODlv0RvOP"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1729172697510
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_1016588812108610"
                    },
                    {
                        "download_result": {
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\Thiago\\dwhelper"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Congresso de Redes Sociais Hotmart Club-02.mp4"
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\Thiago\\dwhelper\\Congresso de Redes Sociais Hotmart Club-02.mp4"
                            },
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.congresso-redessociais.com/en/club/congresso-redessociais/products/4530907/content/meLlRzpWen"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1729172761416
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_7692254311937444"
                    },
                    {
                        "download_result": {
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\Thiago\\dwhelper"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Congresso de Redes Sociais Hotmart Club-03.mp4"
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\Thiago\\dwhelper\\Congresso de Redes Sociais Hotmart Club-03.mp4"
                            },
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.congresso-redessociais.com/en/club/congresso-redessociais/products/4530907/content/146a5W3oOd"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1729175972596
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_4210643761035548"
                    },
                    {
                        "download_result": {
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\Thiago\\dwhelper"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Congresso de Redes Sociais Hotmart Club-01.mp4"
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\Thiago\\dwhelper\\Congresso de Redes Sociais Hotmart Club-01.mp4"
                            },
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.congresso-redessociais.com/en/club/congresso-redessociais/products/4530907/content/Me1lZxgj4Y?source=CLASS_MODULES_LIST"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1729177978081
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_3959946686070494"
                    },
                    {
                        "download_result": {
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\Thiago\\dwhelper"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Congresso de Redes Sociais Hotmart Club.mp4"
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\Thiago\\dwhelper\\Congresso de Redes Sociais Hotmart Club.mp4"
                            },
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.congresso-redessociais.com/en/club/congresso-redessociais/products/4530907/content/0OvW08zd4j"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1729178142952
                        }
                    }
                ]
            }
        ]
    },
    "record_download_history": true
}

**Comment by jcfrog** [comment]
Hi, sorry for this, I'll send you an email.
